zerver/views/webhooks/github.py

```diff
@@ -260,6 +260,19 @@ def build_message_from_gitlog(user_profile, name, ref, commits, before, after, u
                       short_ref,
                       after[:7]))
     else:
-        content = build_commit_list_content(commits, short_ref, url, pusher)
+        from zerver.lib.webhooks.git import get_push_commits_event_message
```

Comment:
Yep sure I just published it for reviewing the concept. Before sending it to official review I will fix all of lint issues